RFS/src/testFixtures/java/com/rfs/framework/SearchClusterContainer.java

```diff
@@ -17,60 +18,35 @@
  */
 @Slf4j
 public class SearchClusterContainer extends GenericContainer<SearchClusterContainer> {
-    public static final String CLUSTER_SNAPSHOT_DIR = "/usr/share/elasticsearch/snapshots";
+    public static final String CLUSTER_SNAPSHOT_DIR = "/tmp/snapshots";
```

Comment:
When running on my linux machine the original path caused OS to fail to start because it couldn't access the path.

Reply:
I thought that this path was for the local container, not the host.

Reply:
This is the path on the container, but it looks like it didn't work on OS (OpenSearch) as I saw startup failures.